### PR TITLE
Update cpp-java-python.md

### PR DESCRIPTION
**Pull request by fgr**
**Description**
In the C++ example, it is not necessary to use a pointer to a heap-allocated `World` object. Instead a local variable can be used (which also gets rid of the need to delete the `World` object.

**Related Issues**
None

**Tasks**
Add the list of tasks of this PR.
  - [x] Update C++ example source code in the documentation

**Documentation**
https://cyberbotics.com/doc/guide/cpp-java-python?version=released

### CODE REVIEW

**Review by ad-daniel — APPROVED** [review]
Indeed, thank you!

**Comment by omichel** [comment]
@fgr: Thank you for this contribution! I sent you a GitHub invite to become a member of the Webots committers, so that you should be able to merge this PR yourself after you accept the invite.

**Comment by fgr** [comment]
@omichel Thank you for inviting me to the organization. I merged this PR.

**Pull request merged**
